packages/coordinator/src/components/Header/CoordinatorHeader.tsx

```diff
@@ -13,23 +13,30 @@ interface CoordinatorHeaderProps {
 export default function CoordinatorHeader(props: CoordinatorHeaderProps) {
   const history = useHistory();
 
-  const navigate = (screen: CoordinatorScreen) => () =>
+  const [currentTab, setCurrentTab] = useState("home");
+
+  const navigate = (screen: CoordinatorScreen) => () => {
```

Comment:
The problem with this is that it won't change when users hit "back" or manually navigate to a different screen.

Is there a way to use the history hook to know what page we are in?

Reply:
fixing it, initially started doing it with useLocation, then changed to a state.
will update soon